src/sagemaker/mxnet/README.rst

```diff
@@ -599,13 +599,16 @@ The code executed from your main guard needs to:
 3. Save the model
 
 Hyperparameters will be passed as command-line arguments to your training script.
-In addition, the locations for finding input data and saving the model and output data will need to be defined.
+In addition, the locations for finding input data and saving the model and output data will be provided as environment variables rather than as arguments to a function.
```

Comment:
Suggestion:

In addition, you specify the locations of input data and where to save the model artifacts and output data as environment variables in the container, rather than as arguments to a function.

Reply:
the container provides this info to the user, not the other way around. Would this be clearer?

In addition, the container will define the locations of input data and where to save the model artifacts and output data as environment variables rather than passing that information through `train`.

Reply:
Minor change:

...rather than passing that information as arguments to the ``train`` function.